### DISCUSSION TITLE
Can't install BthPS3

### DISCUSSION THREAD

**Discussion by nickwzxx** [nefarius/BthPS3#77] [Support]
Hello everyone! I'm having a trouble with installing this driver. Can anyone help me please!? I keep getting a lot of errors and here is the full story

So, at first, my bluetooth adaptor is Comfast CF-B01, and it shows up in Windows as Generic Bluetooth Radio.
![photo_2023-06-01_19-02-36](https://github.com/ViGEm/BthPS3/assets/116093195/69651976-452e-4236-926c-3361c303db23)

The device is shown as this, and it should be supported
![photo_2023-06-01_19-05-05](https://github.com/ViGEm/BthPS3/assets/116093195/0083f1ed-cee0-4b51-bb6e-bc60b9b72e97)

After BthPS3 installation, I'm getting Code 10 error and it's not working at all
![photo_2023-06-01_19-06-34](https://github.com/ViGEm/BthPS3/assets/116093195/8728e2d1-2ab1-4fb3-9c06-827b66a1559e)

So I decided to read guide and found this command and tried it:
Remove-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Class\{e0cbf06c-cd8b-4647-bb8a-263b43f0f974}' -Name 'LowerFilters'

After this command, my bluetooth starts to work again, but BthPS3 driver won't.
![photo_2023-06-01_19-09-53](https://github.com/ViGEm/BthPS3/assets/116093195/a1b2bc03-70be-461d-8e15-5eee795be971)

I tried to reinstall it again but the story repeats and nothing really helps.

I provided as much information as I could, waiting for help. Thanks in advance!

**Comment by nickwzxx** [comment]
Found one more thing to share

![photo_2023-06-01_19-20-45](https://github.com/ViGEm/BthPS3/assets/116093195/f44e077e-b9e0-4941-a452-7bc301e043f0)

Device BTHENUM\{1cb831ea-79cd-4508-b0fc-85f7c85ae8e0}_LOCALMFG&0000\7&11c1e933&0&000000000000_00000002 had a problem starting.

Driver Name: oem15.inf
Class Guid: {e0cbf06c-cd8b-4647-bb8a-263b43f0f974}
Service: BthPS3
Lower Filters: 
Upper Filters: 
Problem: 0x0
Problem Status: 0xC00000E5

**Comment by lehh** [comment]
I'm having the exact same problem. Also sharing the firmware version in my case.
![image](https://github.com/nefarius/BthPS3/assets/13219971/99a243f0-5450-4782-aa1f-a316f0638ec2)

The hardware ID is the same. It's a BASEUS Bluetooth 5.1 adapter.

I'm running on:
```
Edition	Windows 10 Pro
Version	22H2
OS build	19045.3803
```

**Comment by andya34** [comment]
i have the same hid and same bluetooth manufecturer as you and having the exact same problem. guess having the same hid in the document doesn't mean it will work flawlessly. maybe it's an issue with bt 5.1 dongles?

**Comment by ChangoD123** [comment]
> Hello everyone! I'm having a trouble with installing this driver. Can anyone help me please!? I keep getting a lot of errors and here is the full story
> 
> So, at first, my bluetooth adaptor is Comfast CF-B01, and it shows up in Windows as Generic Bluetooth Radio. ![photo_2023-06-01_19-02-36](https://private-user-images.githubusercontent.com/116093195/242658739-69651976-452e-4236-926c-3361c303db23.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ5MTc3NTgsIm5iZiI6MTcwNDkxNzQ1OCwicGF0aCI6Ii8xMTYwOTMxOTUvMjQyNjU4NzM5LTY5NjUxOTc2LTQ1MmUtNDIzNi05MjZjLTMzNjFjMzAzZGIyMy5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDExMFQyMDEwNThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZGU1YzdlZDE3YWRmYjRiZWNkYTRiZWU1MzlkZmI4ZTEzNDViMWEwMDIwZTBkMTUzZTE2MTk5YmRjY2NlNjYxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.vLZS7-hpl_klksj0vG1aMIMFx1fdCOsUXXhuMzkcG7M)
> 
> The device is shown as this, and it should be supported ![photo_2023-06-01_19-05-05](https://private-user-images.githubusercontent.com/116093195/242659196-0083f1ed-cee0-4b51-bb6e-bc60b9b72e97.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ5MTc3NTgsIm5iZiI6MTcwNDkxNzQ1OCwicGF0aCI6Ii8xMTYwOTMxOTUvMjQyNjU5MTk2LTAwODNmMWVkLWNlZTAtNGI1MS1iYjZlLWJjNjBiOWI3MmU5Ny5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDExMFQyMDEwNThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wODc2OGFkNjU0N2E0M2E3N2IxYzVlODM2NzFiNjc2ZGE5NzI5YTA1NDhiOWMzMzk0OGNiZTNmYjYwMzI0ZjkwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.vZ8EMRupJi44j6SAg0uATkH66mkVIZOeMocyu5lcc-k)
> 
> After BthPS3 installation, I'm getting Code 10 error and it's not working at all ![photo_2023-06-01_19-06-34](https://private-user-images.githubusercontent.com/116093195/242659627-8728e2d1-2ab1-4fb3-9c06-827b66a1559e.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ5MTc3NTgsIm5iZiI6MTcwNDkxNzQ1OCwicGF0aCI6Ii8xMTYwOTMxOTUvMjQyNjU5NjI3LTg3MjhlMmQxLTJhYjEtNGZiMy05YzA2LTgyN2I2NmExNTU5ZS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDExMFQyMDEwNThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NGU4MmJkN2M3NjFkN2Y2OWQyZjBlYjkzYThkMzVkODM5Njg4ODE3YzVkYTU1NmMwOGY0MDJkNGZmMDdkYTQ0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.HB409LJCaNH4jX2aEHKOZFqab6XHsiBBLMXKkedcRF0)
> 
> So I decided to read guide and found this command and tried it: Remove-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Class{e0cbf06c-cd8b-4647-bb8a-263b43f0f974}' -Name 'LowerFilters'
> 
> After this command, my bluetooth starts to work again, but BthPS3 driver won't. ![photo_2023-06-01_19-09-53](https://private-user-images.githubusercontent.com/116093195/242660449-a1b2bc03-70be-461d-8e15-5eee795be971.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ5MTc3NTgsIm5iZiI6MTcwNDkxNzQ1OCwicGF0aCI6Ii8xMTYwOTMxOTUvMjQyNjYwNDQ5LWExYjJiYzAzLTcwYmUtNDYxZC04ZTE1LTVlZWU3OTViZTk3MS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDExMFQyMDEwNThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZWZiOWM5NmNjY2ZkYTlkYTQ1NTA3MWQ1YjBhYjkzOTVhZTFhMTNhZDFiYjBkNjdiOGE0N2FjYjE1ZDU2ZjYwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.NMIIW0LkLerNRMvI92aEBhhY4NEeMwnmECEkqTYB04E)
> 
> I tried to reinstall it again but the story repeats and nothing really helps.
> 
> I provided as much information as I could, waiting for help. Thanks in advance!

I have the same problem, how did you run the code Remove-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Class{e0cbf06c-cd8b-4647-bb8a-263b43f0f974}' -Name 'LowerFilters' to get your bluetooth working?
